tests/rados/test_bug_fixes.py

```diff
@@ -0,0 +1,299 @@
+                return 0
+
+            daemon_list = ["mon", "mgr", "osd", "rgw", "mds"]
+            for daemon in daemon_list:
+                # retrieve ceph orch service for the daemon
+                orch_services = rados_obj.list_orch_services(service_type=daemon)
+                if not orch_services:
+                    log.info(f"No Orch services found for the daemon: {daemon}")
+                    log.info("Continuing to the next daemon in the list")
+                    continue
+
+                log.info(f"List of orch services for daemon {daemon}: {orch_services}")
+
+                # log daemon processes for respective orch services
+                for service in orch_services:
+                    orch_ps = client_node.exec_command(
+                        cmd=f"ceph orch ps --service_name {service} --refresh",
+                        client_exec=True,
+                    )
+                    log.info(
+                        f"Orch processes for orch service {service}: \n\n {orch_ps}"
+                    )
+
+                for service in orch_services:
+                    orch_ps_json = rados_obj.run_ceph_command(
+                        cmd=f"ceph orch ps --service_name {service} --refresh",
+                        client_exec=True,
+                    )
+                    rand_orch_ps = random.choice(orch_ps_json)
+                    log.info(
+                        f"Chosen Orch process's dameon name: {rand_orch_ps['daemon_name']}"
+                    )
+                    log.info(
+                        f"Chosen Orch process's daemon id: {rand_orch_ps['daemon_id']}"
+                    )
+                    log.info(
+                        f"Chosen Orch process's daemon type: {rand_orch_ps['daemon_type']}"
+                    )
+
+                    if daemon == "rgw":
+                        config_get_op = mon_obj.get_config(
+                            daemon_name=f"client.{rand_orch_ps['daemon_name']}"
+                        )
+                        config_show_op = mon_obj.show_config(
+                            daemon="client", id=rand_orch_ps["daemon_name"]
+                        )
+                    else:
+                        config_get_op = mon_obj.get_config(
+                            daemon_name=rand_orch_ps["daemon_name"]
+                        )
+                        config_show_op = mon_obj.show_config(
+                            daemon=daemon, id=rand_orch_ps["daemon_id"]
+                        )
+
+                    log.info(
+                        f"ceph config get output for the daemon {rand_orch_ps['daemon_name']}: "
+                        f"\n\n {config_get_op}"
+                    )
+                    log.info(
+                        f"ceph config show output for the daemon {rand_orch_ps['daemon_name']}: "
+                        f"\n\n {config_show_op}"
```

Comment:
Is there a need to validate if both outputs are producing same results? Or have we seen they do not match in some cases?